include/circt/Dialect/LLHD/IR/ExtractOps.td

```diff
@@ -101,3 +101,118 @@ def LLHD_DextsOp : LLHD_Op<"dexts", [
     unsigned getTargetWidth() { return getLLHDTypeWidth(target().getType()); }
   }];
 }
+
+def LLHD_ExtfOp : LLHD_Op<"extf", [
```

Comment:
prefer complete words.  IS this "Extend Float?" "Extract field?" "External foobar?" "Example Tensorflow?"